spec/requests/comments_spec.rb

```diff
@@ -270,4 +270,24 @@
   describe "PATCH /comments/:comment_id/unhide" do
     include_examples "PATCH /comments/:comment_id/hide or unhide", path: "unhide", hidden: "false"
   end
+
+  describe "DELETE /comments/:comment_id" do
+    before { sign_in user }
+
+    it "deletes a comment if the article is still present" do
+      delete "/comments/#{comment.id}"
+
+      expect(Comment.find_by(id: comment.id)).to be_nil
+      expect(flash[:notice]).to eq("Comment was successfully deleted.")
+    end
+
+    it "deletes a comment if the article has been deleted" do
+      article.destroy!
+
+      delete "/comments/#{comment.id}"
+
+      expect(Comment.find_by(id: comment.id)).to be_nil
+      expect(flash[:notice]).to eq("Comment was successfully deleted.")
```

Comment:
And same here: "I think we should test for proper redirection here instead of the flash message."